messaging-client-databus/src/main/java/com/inmobi/messaging/consumer/databus/AbstractMessagingDatabusConsumer.java

```diff
@@ -58,6 +58,7 @@
   protected final Map<PartitionId, PartitionReader> readers =
       new HashMap<PartitionId, PartitionReader>();
 
+  protected final Set<PartitionReader> startedReaders = new HashSet<PartitionReader>();
```

Comment:
can we move this to DatabusConsumer if possible and rename it to startedCollectorReaders?